docs/overview/component/filter-gateway.md

```diff
@@ -0,0 +1,22 @@
+# Vald Filter Gateway
+
+Vald Filter Gateway is the component to communicate user defined ingress / egress filter components with requests.
+
+This page introduces the overview of Vald Filter Gateway
+
+## Responsibility
+
+Vald Filter Gateway is the optional component in the Vald cluster.
+
+It is implemented for available of user-defined filtering.
+The responsibilities is bypass between the Vald cluster and user defined ingress filter (or egress filter) components
```

Comment:
```suggestion
The responsibility is bypass between the Vald cluster and user-defined ingress filter (or egress filter) components.
```
My Grammarly said responsibility is an uncountable noun and user-defined is the correct user defined.
If bypass is a noun, that is correct.